inventory/libvirt/hosts/libvirt_generic.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
```

Comment:
As for other dynamic inventory scripts `inventory/*/hosts/*.py`, I would rather have
`#!/usr/bin/env python2`
to be sure that `python2` is used even on distributions where `python` is symlinked to `python3`.